### ISSUE TITLE
Style/GuardClause triggered for one-line conditions with `elsif`

### ISSUE THREAD

**Issue by AlexWayfer** [bbatsov/rubocop#4108]
## Expected behavior

No warnings.

## Actual behavior

```
test.rb:3:5: C: Use a guard clause instead of wrapping the code inside a conditional expression.
    if element.is_a?(Symbol)
    ^^
```

## Steps to reproduce the problem

```ruby
def foo
  [1, 2, :a, 3, 'b'].find do |element|
    if element.is_a?(Symbol)
      break 'Symbol found '
    elsif element.is_a?(String)
      break 'String found'
    end
  end || 'No Symbol or String found'
end
```

### 1

```ruby
def foo
  [1, 2, :a, 3, 'b'].find do |element|
    if element.is_a?(Symbol)
      break 'Symbol found ' \
        'in array' # <= here
    elsif element.is_a?(String)
      break 'String found'
    end
  end || 'No Symbol or String found'
end
```

### 2

```ruby
def foo
  [1, 2, :a, 3, 'b'].find do |element|
    if element.is_a?(Symbol)
      break 'Symbol found '
    end # <= here
    if element.is_a?(String)
      break 'String found' \
        'in array' # <= and here
    end
  end || 'No Symbol or String found'
end
```

## RuboCop version

```
$ rubocop -V
0.47.1 (using Parser 2.4.0.0, running on ruby 2.4.0 x86_64-linux)
```

**Comment by AlexWayfer** [comment]
Resolved:

```ruby
def foo
  [1, 2, :a, 3, 'b'].find do |element|
    if element.is_a?(Symbol)
      break 'Symbol found '
    end
    next unless element.is_a?(String)
    break 'String found'
  end || 'No Symbol or String found'
end
```

Just unintuitive place of the highlight (why not `elsif`?)
